template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/core/CoreConfigProvider.java

```diff
@@ -91,7 +91,8 @@ public List<ApiClusterTemplateConfig> getServiceConfigs(CmTemplateProcessor temp
     }
 
     private boolean isSDXOptimizationNeeded(TemplatePreparationObject source) {
```

Comment:
maybe worth moving this method into a static method in a separate class? This would allow to reduce code duplication.

Reply:
no static please